src/shapes/MoviesListShape.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+export const MoviesListShape = {
+  list: PropTypes.arrayOf(
+    PropTypes.shape = {
+      title: PropTypes.string.isRequired,
+      description: PropTypes.string,
+      imgUrl: PropTypes.string.isRequired,
+      imdbUrl: PropTypes.string.isRequired,
+    },
```

Comment:
you can import MovieCardShape here or create a general Shape.js file with all the shapes inside of it